zimbra/src/main/java/fr/openent/zimbra/service/impl/ReturnedMailService.java

```diff
@@ -327,11 +330,13 @@ private void deleteMailFromZimbra(List<String> ids, UserInfos user, Handler<Eith
                             if (event.isRight()) {
                                 handler.handle(new Either.Right<>(event.right().getValue()));
                             } else {
-                                handler.handle(new Either.Left<>("[Zimbra] getReturnedMailsInfos : Error while deleting mails : " + event.left().getValue()));
+                                log.error(String.format("[Zimbra@%s::deleteMailFromZimbra] Error when deleting mails: %s", this.getClass().getSimpleName(), event.left().getValue()));
+                                handler.handle(new Either.Left<>(String.format("[Zimbra@%s::deleteMailFromZimbra] Error when deleting mails: %s", this.getClass().getSimpleName(), event.left().getValue())));
```

Comment:
Pas besoin d'envoyer le même log dans le handler, mets seulement le getValue
Les logs avec le format etc est plus pour les logs de vertx

```suggestion
                                                               handler.handle(event.left().getValue());

```